### PR TITLE
Added code to execute queries to make ybvoyager_metadata and the tabl…

### PR DESCRIPTION
**Pull request by ShivanshGahlot**
…e inside it  using a single new connection

### CODE REVIEW

File: yb-voyager/cmd/importData.go [hunk 1]
```diff
@@ -358,16 +358,32 @@ func createVoyagerSchemaOnTarget(connPool *tgtdb.ConnectionPool) error {
...
-		err := connPool.WithConn(func(conn *pgx.Conn) (bool, error) {
-			_, err := conn.Exec(context.Background(), cmd)
-			return false, err
-		})
-		if err != nil {
-			return fmt.Errorf("create ybvoyager schema on target: %w", err)
+
+	maxAttempts := 12
+	currAttempt := 0
+	err := connPool.WithConn(func(conn *pgx.Conn) (bool, error) {
```
Comment (amit-yb) [inline yb-voyager/cmd/importData.go:364]: Do not use `connPool`. We used `newTargetConn()` to make it easier to implement retries.
```
outer:
for _, cmd := range cmds {
  for attempt := 1; attempt <= 12; attempt++ {
    log.Infof(...)
    if conn == nil {
      conn = newTargetConn()
    }
    _, err = conn.Exec(...)
    if err == nil {
      continue outer;
    }
    log.Warnf("Error while running [%s] attempt %d: %s", cmd, attempt, err)
    time.Sleep(5 * time.Second)
    conn.Close()
    conn = nil
  }
  if err != nil {
    utils.ErrExit("...")
  }
}
```

Comment (ShivanshGahlot) [reply yb-voyager/cmd/importData.go]: 👍

File: yb-voyager/cmd/importData.go [hunk 2]
```diff
@@ -358,16 +358,31 @@ func createVoyagerSchemaOnTarget(connPool *tgtdb.ConnectionPool) error {
...
+			log.Warnf("Error while running [%s] attempt %d: %s", cmd, attempt, err)
+			time.Sleep(5 * time.Second)
+			conn.Close(context.Background())
+			conn = nil
+		}
 		if err != nil {
 			return fmt.Errorf("create ybvoyager schema on target: %w", err)
 		}
 	}
+
```
Comment (amit-yb) [inline yb-voyager/cmd/importData.go:385]: Nit:
```
if conn != nil {
  conn.Close(context.Background())
}
```

**Review by amit-yb — APPROVED** [review]
LGTM with a minor comment.

**Pull request merged**